### DISCUSSION TITLE
Make easier to find the highlights on page

### DISCUSSION THREAD

**Discussion by carmolim** [omnivore-app/omnivore#2524] [Ideas]
One feature I like from Raindrop and Hypothesis is to mark on the scroll bar where the highlight were made on page: 

**Raindrop**
![image](https://github.com/omnivore-app/omnivore/assets/2863161/1b9ec8d5-f27f-42ad-a59e-d3e646b892cf)

**Hypothesis**
![image](https://github.com/omnivore-app/omnivore/assets/2863161/7cababd8-bf83-40d3-9892-d41981af30a8)

I think that this would be a great addition to Omnivore.

**Comment by jacksonh** [comment]
Thanks, going to show this one to our designer. I like how it shows "upcoming"

one thing that might give the same effect in Omnivore is if you use the `j`, `k` keys you can navigate to each highlight.
